crates/rooch/src/commands/move_cli/commands/integration_test.rs

```diff
@@ -340,3 +339,22 @@ impl IntegrationTest {
         Ok(())
     }
 }
+
+fn parse_account_arg(account: String) -> Result<AccountAddress, RoochError> {
```

Comment:
We should use the `WalletContext::parse_account_arg` https://github.com/rooch-network/rooch/blob/main/crates/rooch-rpc-client/src/wallet_context.rs#L73-L75